boards/adafruit/feather_esp32/board.yml

```
@@ -0,0 +1,11 @@
+board:
+  name: adafruit_feather_esp32
+  full_name: Adafruit ESP32 Feather
```

Nit: all the other supported Feather boards are named "Adafruit Feather XXX" 

```suggestion
  full_name: Adafruit Feather ESP32
```

---

That's fair. I originally named it based on the adafruit.com listing but now I see all ESP32 feathers there now use the "Adafruit ESP32... Feather" convention. It's definitely better to have naming consistency here, so I've updated the name in the yml and dts files :)